.ddev/config.yaml

```
@@ -0,0 +1,35 @@
+name: astrosite
+type: php
+docroot: dist
+webserver_type: nginx-fpm
+additional_hostnames: []
+additional_fqdns: []
+performance_mode: none
+use_dns_when_possible: true
+web_environment: []
+nodejs_version: "18"
+
+# 4321 is Astro Vite default
+web_extra_exposed_ports:
+- name: astro-dev
+  container_port: 4321
+  http_port: 4322
+  https_port: 4321
+
+# The extra -- is a Vite item
+# https://github.com/vitejs/vite/discussions/3396
+web_extra_daemons:
+  - name: astro-dev-deamon
```

Might as well spell it as expected. 
```suggestion
  - name: astro-dev-daemon
```

---

Sounds good